### PR TITLE
add py-flameprof

### PR DESCRIPTION
**Pull request by haampie**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-flameprof/package.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+    homepage = "https://github.com/baverman/flameprof/"
+    pypi     = "flameprof/flameprof-0.4.tar.gz"
+
+    maintainers = ['haampie']
+
+    version('0.4', sha256='dbc86d4190cbbba624f1e0a40f44d9db96138e27534d83c8ef42d420857875a3')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-uwsgi', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-flameprof/package.py]: Where do you see this dependency listed?

Comment (haampie) [reply var/spack/repos/builtin/packages/py-flameprof/package.py]: rqrmnts.txt or something

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-flameprof/package.py]: There isn't one, that's why I'm confused.

Comment (haampie) [reply var/spack/repos/builtin/packages/py-flameprof/package.py]: ah okay, https://github.com/baverman/flameprof/blob/master/reqs.txt is what i got

Comment (haampie) [reply var/spack/repos/builtin/packages/py-flameprof/package.py]: but it's not in the tarball indeed...

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-flameprof/package.py]: Even if it was in the tarball I would probably ignore it, it isn't used by setup.py.

Comment (haampie) [reply var/spack/repos/builtin/packages/py-flameprof/package.py]: Ok, dropped it

**Review by adamjstewart — DISMISSED** [review]

File: var/spack/repos/builtin/packages/py-flameprof/package.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFlameprof(PythonPackage):
+    """Flamegraph generator for python’s cProfile stats."""
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-flameprof/package.py]:
```suggestion
    """Flamegraph generator for python's cProfile stats."""
```
Unicode characters like smart quotes aren't valid in Python 2, hence the unit test failures.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**